documentation/configuringDynaflowLauncher/DynaflowLaunchersInputFiles.tex

```diff
@@ -159,9 +161,10 @@ \subsection{Configuration file}
 \caption{Chosen Outputs depending on the buildtype and the simulation kind : outputs chosen by default are \textcolor{green}{green} and optional inputs are \textcolor{gray}{gray}}
 \end{table}
 
-The systematic analysis parameters are described below:
+The \textit{StartingPointMode} indicates the starting state of the simulation. If it's warm, the simulation starts from a previous simulation state. If it's flat, the simulation starts from scratch.
 
 \begin{table}[H]
+The systematic analysis parameters are described below:
```

Comment:
Why is it in the table?

Reply:
I moved it down in the table to keep this phrase with the table below.

Inside the table :
![without](https://user-images.githubusercontent.com/108793615/197477647-d137251f-a7c3-42f4-963d-118762098aa5.png)

Outside the table :
![with](https://user-images.githubusercontent.com/108793615/197477644-53180f54-b8bb-4576-9525-12331c6e7ac1.png)